src/config.coffee

```diff
@@ -32,6 +32,11 @@ module.exports = {
         type : 'boolean'
         default : true
         description : "When beautifying on save, use the entire file, even if there is selected text in the editor. Important: The `beautify on save` option for the specific language must be enabled for this to be applicable. This option is not `beautify on save`."
+      beautifyFileExtensions :
+        title: "Do not Beautify these extensions"
+        type : 'string'
```

Comment:
Change type `string` to `array`. See https://atom.io/docs/api/v1.6.0/Config for details.
For example:
```coffeescript
config:
  excludeFileExtensions:
    type: 'array'
    default: ['.min.js', '.min.css']
    items:
      type: 'string'
```